src/libs/actions/PersonalDetails.js

```diff
@@ -84,7 +84,6 @@ function getMaxCharacterError(isError) {
  * @return {Object}
  */
 function formatPersonalDetails(personalDetailsList) {
-    Timing.start(CONST.TIMING.PERSONAL_DETAILS_FORMATTED);
     const formattedResult = {};
 
     // This method needs to be SUPER PERFORMANT because it can be called with a massive list of logins depending on the policies that someone belongs to
```

Comment:
This comment makes me think that this could be a useful timing event after all. Why not create a graph instead of removing the event altogether?

Reply:
If no one is using it, then it's not useful. If someone thinks it is useful in the future, they can add it back.

Or said differently: does a metric make a sound if there's no one around to look at the graph? :trollface: